mithril-common/src/crypto_helper/cardano/key_certification.rs

```diff
@@ -0,0 +1,317 @@
+use crate::crypto_helper::cardano::{OpCert, ParseError, SerDeShelleyFileFormat};
```

Comment:
I think the `.rs` files should include a small description to increase the code readability.  This suggestion holds for each file that does not include descriptions.

```suggestion
//! API for mithril key certification. 
//! Includes the wrappers for StmInitializer and KeyReg, and ProtocolRegistrationErrorWrapper. 

use crate::crypto_helper::cardano::{OpCert, ParseError, SerDeShelleyFileFormat};
```